elements-fun/src/transaction.rs

```diff
@@ -327,6 +327,30 @@ pub struct PegoutData<'txo> {
     pub extra_data: Vec<&'txo [u8]>,
 }
 
+/// These constants have been reverse engineered through the following transactions:
+///
+/// https://blockstream.info/liquid/tx/a17f4063b3a5fdf46a7012c82390a337e9a0f921933dccfb8a40241b828702f2
+/// https://blockstream.info/liquid/tx/d12ff4e851816908810c7abc839dd5da2c54ad24b4b52800187bee47df96dd5c
+/// https://blockstream.info/liquid/tx/47e60a3bc5beed45a2cf9fb7a8d8969bab4121df98b0034fb0d44f6ed2d60c7d
```

Comment:
Did you see that these 3 are `p2sh` transactions? 

An improvement could be to do 3 transactions with the mentioned combinations and record the vbytes.

Reply:
Note: This code was only moved in this PR, not added.

The majority of the size comes from the range proof which is almost a kilobyte. I'd expect the size of the script to therefore play a negligible role.